Phonebook.Frontend/src/app/shared/components/navigation/navigation.component.scss

```diff
@@ -1,6 +1,27 @@
 @import 'media-queries';
 @import '~@angular/material/theming';
 
+@keyframes changeColor {
+  0% {
+    color: #ff0000;

```

Comment:
That's got nothing to do with the wrong information button?

Reply:
my b 
I played with the css and forget to undo befor stage my changes